.github/workflows/npm_napi_publish.yml

```
@@ -52,7 +52,7 @@ jobs:
             arch: x86_64-unknown-linux-gnu
             ext: ""
             docker_image: ""
-            additional_packages: "librust-atk-dev libgtk-3-dev libglib2.0-dev libasound2-dev"
+            additional_packages: "librust-atk-dev libgtk-3-dev libglib2.0-dev alsa-lib-dev"
```

not needed, it's the correct package on ubuntu
```suggestion
            additional_packages: "librust-atk-dev libgtk-3-dev libglib2.0-dev libasound2-dev"
```